pages/_document.js

```diff
@@ -36,25 +37,37 @@ export default class MyDocument extends Document {
     return (
       <Html dir="ltr">
         <Head>
-          {/* Global Site Tag (gtag.js) - Google Analytics */}
+          <Fragment></Fragment>
           <script
             async
             src={`https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}`}
           />
```

Comment:
I believe this `<script>` element is part of what should be removed for this piece of work.

Reply:
This has been removed